src/client_lib/pegasus_client_impl.h

```diff
@@ -265,12 +265,14 @@ class pegasus_client_impl : public pegasus_client
 
         pegasus_scanner_impl(::dsn::apps::rrdb_client *client,
                              std::vector<uint64_t> &&hash,
-                             const scan_options &options);
+                             const scan_options &options,
+                             bool validate_partition_hash);
```

Comment:
What about move validate_partition_hash into scan_options?

Reply:
Scan_options is a structure for user-defined options, `validate_partition_hash` should not be aware for user. For example, `get_scanner` interface is like:
```
int pegasus_client_impl::get_scanner(const std::string &hash_key,
                                     const std::string &start_sort_key,
                                     const std::string &stop_sort_key,
                                     const scan_options &options,
                                     pegasus_scanner *&scanner);
```